.azure-pipelines/client.yml

```diff
@@ -50,6 +46,35 @@ jobs:
     displayName: 'Publish Artifacts'
     condition: succeededOrFailed()
 
+- job: 'Analyze'
+  dependsOn:
+    - 'Build_Target_Packages'
+  
+  pool:
+    vmImage: 'vs2017-win2016' 
+    
+  steps:
+  
+  - task: UsePythonVersion@0
+    displayName: 'Use Python 3.6'
+    inputs:
+      versionSpec: 3.6
```

Comment:
Please move to variable, and replace this hardcoded version number (and the earlier hardcoded version) with the variable.